### PR TITLE
Adding the support/license related content and some minor improvements.

### PR DESCRIPTION
**Pull request by Serdaro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,18 +66,24 @@
...
 
-This document explains Node.js client for Hazelcast which uses Hazelcast's Open Client Protocol 1.6. This client works with Hazelcast 3.6 and higher.
+You can see the release notes for each Node.js client release on the [Releases](https://github.com/hazelcast/hazelcast-nodejs-client/releases) page of this repository.	
 
-**Hazelcast** is a clustering and highly scalable data distribution platform. With its various distributed data structures, distributed caching capabilities, elastic nature and more importantly with so many happy users, Hazelcast is a feature-rich, enterprise-ready and developer-friendly in-memory data grid solution.
 
 # 1. Getting Started
 
-This chapter explains all the neccessary things to start using Hazelcast Node.js Client including basic Hazelcast IMDG, IMDG and client
+This chapter explains all the neccessary things to start using Hazelcast Node.js Client including basic Hazelcast IMDG and client
```
Comment (sancar) [inline README.md]: `neccessary` -> `necessary`

File: README.md [hunk 2]
```diff
@@ -138,12 +149,14 @@ Find more information about `hazelcast-member` tool at https://github.com/hazelc
 Refer to the official [Hazelcast IMDG Reference Manual](http://docs.hazelcast.org/docs/3.10.4/manual/html-single/index.html#getting-started) for more information regarding starting clusters.
```
Comment (sancar) [inline README.md]: I think we need to be more consistent when using links to http://docs.hazelcast.org/docs/
Some usages link to `3.10.4` / `3.10.5`,  some usages refer to `latest`.

File: README.md (hunk 1 above)
Comment (Serdaro) [reply README.md]: 👍

File: README.md [hunk 3]
```diff
@@ -1671,19 +1692,25 @@ npm test
...
+You can use the following channels for your questions and development/usage issues:
+
+* This repository by opening an issue.
+* Hazelcast Node.js client channel on Gitter: 
+[![Join the chat at https://gitter.im/hazelcast-incubator/hazelcast-nodejs-client](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/hazelcast-incubator/hazelcast-nodejs-client?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+* Our Google Groups directory: https://groups.google.com/forum/#!forum/hazelcast
+* Stackoverflow: https://stackoverflow.com/questions/tagged/hazelcast
 
-Please join the mail group if you are interested in using or developing Hazelcast.
+# 9. Contributing
```
[the remark this answers is not in the document]
Comment (Serdaro) [reply README.md]: 👍

File: README.md (hunk 2 above)
Comment (Serdaro) [reply README.md]: Made them all "latest"

**Pull request merged**